### PR TITLE
Disable flake8 error E722 (fix #471)

### LINKED ISSUE #471: All CI is failing due to new, unexpected flake8 failures
https://travis-ci.org/rec/BiblioPixel/jobs/291490137

    ./bibliopixel/animation/collection.py:75:9: E722 do not use bare except'
    ./bibliopixel/colors/names.py:67:9: E722 do not use bare except'
    ./bibliopixel/colors/names.py:74:5: E722 do not use bare except'
    ./bibliopixel/colors/names.py:86:5: E722 do not use bare except'
    ./bibliopixel/colors/names.py:101:9: E722 do not use bare except'
    ./bibliopixel/colors/names.py:107:9: E722 do not use bare except'

...etc

Why has this suddenly changed?   Sending out a review...

### PR DESCRIPTION
**Pull request by rec**

### CODE REVIEW

File: bibliopixel/main/all_pixel.py [hunk 1]
```diff
@@ -23,11 +23,11 @@
 ]
 
 
-def O(msg):
+def O(msg):  # noqa: E743
```
Comment (adammhaile) [inline bibliopixel/main/all_pixel.py]: This doesn't actually work in flake8 last I checked, it will instead disable flake8 for the entire file.

Comment (adammhaile) [reply bibliopixel/main/all_pixel.py]: Wait... actually, check your version of flake8... 3.1 and up should work.

Comment (rec) [reply bibliopixel/main/all_pixel.py]: Eh, it's not an issue as I'll get rid of those entirely in a subsequent change.

**Comment by rec** [comment]
But I don't get it.  Why would my flake8 version have anything to do with this?

We just started getting Travis breakage on all this untouched code...

**Comment by adammhaile** [comment]
Cool :) +1

**Comment by adammhaile** [comment]
That I have no idea. Blame travis. noqa used to not work in Flake8, that
was all I was getting at. It does now. Carry on.

**Pull request merged**